packages/labs/ssr/README.md

```diff
@@ -43,15 +51,99 @@ export const renderTemplate = (someData) => {
 // Example: server.js:
 
 import {renderModule} from 'lit-ssr/lib/render-module.js';
-// Execute the above `renderTemplate` in a sandboxed VM with a minimal DOM shim
+
+// Execute the above `renderTemplate` in a separate VM context with a minimal DOM shim
 const ssrResult = await (renderModule(
-  './render-template.js',  // Module to load in SSR sandbox
+  './render-template.js',  // Module to load in VM context
   import.meta.url,         // Referrer URL for module
   'renderTemplate',        // Function to call
   [{some: "data"}]         // Arguments to function
 ) as Promise<Iterable<unknown>>);
 
-for (let text of ssrResult) {
-  console.log(text);
-}
+// ...
+
+context.body = Readable.from(ssrResult);
 ```
+
+## Client usage
+
+### Hydrating lit templates
+
+In order to "hydrate" lit templates, which is the process of having lit re-associate the expressions of a lit template with which nodes they should update in the DOM, the `experimental-hydrate` module is provided in the `lit` package. Prior to updating a server-rendered container using `render`, you should first call `hydrate` on that container using the same template and data that was used to render on the server:
+
+```js
+import {myTemplate} from './my-template.js';
+import {render, hydrate} from `lit/experimental-hydrate.js`;
+
+// Initial hydration required before render:
+const initialData = /* should be same data used to render on the server */;
+hydrate(myTemplate(initialData), document.body);
+
+// After hydration, render will efficiently update the server-rendered DOM:
+const update = (data) => render(myTemplate(data), document.body);
+```
+
+### Hydrating LitElements
+
+When `LitElement`s are server rendered, their shadow root contents are emitted inside a `<template shadowroot>`, also known as a [Declarative Shadow Root](https://web.dev/declarative-shadow-dom/), a new browesr feature that shipped in [Chrome 90](https://developer.chrome.com/blog/new-in-chrome-90/#declarative). Declarative shadow roots automatically attach their contents to a shadow root on the template's parent element when parsed. For browsers that do not yet implement declarative shadow root, there is a [`template-shadowroot`](https://github.com/webcomponents/template-shadowroot) ponyfill, described below.
```

Comment:
I might remove the reference to Chrome 90

Reply:
Change it to just "shipping in Chrome"